.gitignore

```diff
@@ -128,3 +128,5 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
```

Comment:
Do we actually need all of these? Looks like pre-optimization. Shall we start small?

Reply:
Also, could we add IDE/editor config files in here?

Reply:
yup the .gitignore absolutely needs to be cleaned